client/tests/webdriver/baseSpecs/userLogin.spec.js

```diff
@@ -7,7 +7,9 @@ describe("Anet home page", () => {
     await (await Home.getBannerDropdown()).click()
     await (await Home.getLogoutButton()).waitForDisplayed()
     await (await Home.getLogoutButton()).click()
-    await Home.waitForLoginForm()
+    const title = await browser.getTitle()
+    await expect(title).to.equal("ANET Logout")
```

Comment:
After this I would check that clicking the **Login** button would take me to the login page, to verify that you're not (still) logged in.